jujugui/static/gui/src/app/assets/javascripts/jujulib/juju.js

```diff
@@ -205,6 +205,26 @@ var module = module;
         // if the macaroon is valid.
         false
       );
+    },
+
+    /**
+      Lists all the templates for cloud credentials for the currently
+      authenticated user.
+
+      @method listTemplates
+      @param callback {Function} A callback to handle errors or accept the data
+          from the request. Must accept an error message or null as its first
+          parameter and the response data as its second.
+    */
+    listTemplates: function(callback) {
+      var url = this.jemURL + '/template';
+      var _listTemplates = function(error, data) {
+        if (error === null) {
+          data = data.templates;
```

Comment:
Is the rest of the data provided not valuable? I understand this is a listTemplates method, I'm just wondering what information we're not showing the user, maybe the full data object could be passed as a third argument to the cb?